common/beerocks/hostapd/include/configuration.h

```diff
@@ -48,6 +48,24 @@ class Configuration {
      */
     bool load();
 
+    /**
+     * @brief set key/value for the given vap
+     * @details set the key/vale for the given vap, either replace or create.
+     * @return 
+     * true - the vap exists, values were set.
+     * false - the given vap was not found
+     */
```

Comment:
```suggestion
    /**
     * @brief Set key/value for the given vap
     * @details Set the key/vale for the given vap, either replace or create.
     * @return true if the vap exists and values were set, false otherwise.
     */
```

note that in case of some kind of failure you would also return false - not only if the VAP doesn't exist.

Reply:
I don't think there is a failing scenario if the `vap` exists. 
The only scenario this function fails is when there is no vap.
And from the application point of view - if you ask to set a value for non-existing vap - indeed this is a _**real**_ error in your flow. It means that you _think_ that this specific vap exists, you want to set one of its values... and a surprise! this vap does not exist - something went completely wrong.